From c7f703c6a81000139bf137d5a84ec02d6019059e Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Wed, 21 Sep 2011 00:35:02 +0800
Subject: ENGR00157155-1 vpu: Add vpu resource name in platform in arch/arm

Add vpu resource name in platform in arch/arm, then vpu driver can
get resource by name.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/plat-mxc/devices/platform-imx_vpu.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/plat-mxc/devices/platform-imx_vpu.c b/arch/arm/plat-mxc/devices/platform-imx_vpu.c
index a3a3927e..b4edb91 100755
--- a/arch/arm/plat-mxc/devices/platform-imx_vpu.c
+++ b/arch/arm/plat-mxc/devices/platform-imx_vpu.c
@@ -123,16 +123,19 @@ struct platform_device *__init imx_add_vpu(
 		{
 			.start = data->iobase,
 			.end = data->iobase + SZ_16K - 1,
+			.name = "vpu_regs",
 			.flags = IORESOURCE_MEM,
 		}, {
 			.start = data->irq_ipi,
 			.end = data->irq_ipi,
+			.name = "vpu_ipi_irq",
 			.flags = IORESOURCE_IRQ,
 		},
 #ifdef CONFIG_SOC_IMX6Q
 		{
 			.start = data->irq_jpg,
 			.end = data->irq_jpg,
+			.name = "vpu_jpu_irq",
 			.flags = IORESOURCE_IRQ,
 		},
 #endif
-- 
1.7.9.5

